lib/matplotlib/axes/_axes.py

```diff
@@ -110,6 +110,77 @@ def inset_locator(ax, renderer):
 
     return inset_locator
 
+def _make_inset_locator_anchored(loc, borderpad, width, height,
+                                bbox_to_anchor, transform, parent):
+    """
+    Helper function to locate inset axes, used in
+    `.Axes.inset_axes`.
+
+    A locator gets used in `Axes.set_aspect` to override the default
+    locations...  It is a function that takes an axes object and
+    a renderer and tells `set_aspect` where it is to be placed.
+
+    Here *rect* is a rectangle [l, b, w, h] that specifies the
+    location for the axes in the transform given by *trans* on the
+    *parent*.
+    """
+    codes = {'upper right':  'NE',
+             'upper left':   'NW',
+             'lower left':   'SW',
+             'lower right':  'SE',
+             'right':        'E',
+             'left':         'W',
+             'bottom':       'S',
+             'top':          'N',
+             'center left':  'W',
+             'center right': 'E',
+             'lower center': 'S',
+             'upper center': 'N',
+             }
```

Comment:
What about the center itself? I.e. loc code `10`.

Reply:
Done.  Thanks for all the help!